suite/tests/CMakeLists.txt

```diff
@@ -5910,10 +5911,9 @@ if (RISCV64)
     code_api|linux.infinite
     code_api|linux.longjmp
     code_api|linux.prctl
-    code_api|linux.sigaction_nosignals
```

Comment:
They're not working on real hardware but they're working on QEMU? How? Would be good to keep the QEMU run if they're indeed working fine there.

Reply:
> if they're indeed working fine there

It's strange, so I'm not sure if they're working as expected, but will add them back once figure out why.